addon/components/sortable-group.js

```diff
@@ -163,10 +187,202 @@ export default Component.extend({
       });
     });
 
-    if (groupModel !== NO_MODEL) {
-      this.onChange(groupModel, itemModels, draggedModel);
-    } else {
+    set(this, 'modelCopy', a(itemModels));
+    // if (groupModel !== NO_MODEL) {
+    //   this.onChange(groupModel, itemModels, draggedModel);
+    // } else {
       this.onChange(itemModels, draggedModel);
+    // }
+  },
+
+  /**
+   * Explanation
+   * 1. `KeyboardReorderMode` is disabled: users can activate it via ENTER or SPACE.
+   * 2. `KeyboardReorderMode` is enabled: users can reorder via UP or DOWN arrow keys. TODO: Expand to more keys, e.g LEFT, RIGHT
+   * 3. `KeyboardReorderMode` is enabled: users can finalize/save the reordering via ENTER or SPACE.
+   * 4. `KeyboardReorderMode` is enabled: users can discard the reordering via ESC.
+   *
+   * @param {Event} evt a HTML event
+   */
+  keyDown(event) {
+    // Note: If handle is specified, we need to target the keyDown on the handle
+    const  { isKeyboardReorderModeEnabled, direction, selectedItem, sortedItems } = this.getProperties('isKeyboardReorderModeEnabled', 'direction', 'selectedItem', 'sortedItems');
+
+    if (!isKeyboardReorderModeEnabled && (isEnterKey(event) || isSpaceKey(event))) {
+      this.prepareKeyboardReorderMode();
+      this.set('isRetainingFocus', true);
+      readDOM(() => {
+        this.element.focus();
+        this.set('isRetainingFocus', false);
+      });
+      // Prevent the default scroll
+      event.preventDefault();
+      return;
+    }
+
+    if (isKeyboardReorderModeEnabled) {
```

Comment:
can we extract it into new method? like `handleKeybordReorder`

Reply:
Yup, will do that in the final PR. I got lazy and just wanted to get something working without spending too much time on making the code nice